src/usher_server.cpp

```diff
@@ -64,13 +59,13 @@ int main(int argc, char** argv) {
             return 1;
     }
 
-
+    //slot for a MAT not in the MAT_list file 
     MAT::Tree loaded_MAT;
     bool loaded_MAT_avail = false;//keep track if loaded_MAT is a new copy of MAT and can be used
     std::string loaded_MAT_name = "";
     
     fs::path p = fs::current_path();
-    p/=arg_dirname;
+    p/=arg_dirname;//get the path to the argument directory
```

Comment:
I prefer not to have comments and code on the same lines. You should shift those comments one line above.